LoopKitUI/Views/DismissibleKeyboardTextField.swift

```diff
@@ -56,6 +54,16 @@ public struct DismissibleKeyboardTextField: UIViewRepresentable {
 
     public func updateUIView(_ textField: UITextField, context: Context) {
         textField.text = text
+        textField.placeholder = placeholder
+        textField.font = font
+        textField.textColor = textColor
+        textField.textAlignment = textAlignment
+        textField.keyboardType = keyboardType
```

Comment:
This group of properties was brought down because they can change after initialization